wikum/website/static/website/js/base.js

```diff
@@ -1996,6 +1996,8 @@ function make_highlight() {
 
 			$('#count_result').text(count + ' comments highlighted');
 
+		}else{
```

Comment:
can you add spaces before and after "else"?

try to imitate the way we're using python elsewhere (and PEP8 guidelines).